src/Common/tests/Performance/PerfTestBase.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.IO;
+
+namespace System
+{
+    /// <summary>Base class for performance test classes</summary>
+    public abstract class PerfTestBase
```

Comment:
The second method is <i>very</i> cumbersome, I agree. That's a good idea though, I think I may switch to to a static util class because I've ran into the issue where there is already a parent class in some test libraries and I've had to choose between the two.

The other option would be just to repeat the utilized methods per class since there isn't many of them, but I'd prefer not to do that if possible :)